core/gui/src/app/workspace/service/ai-assistant/ai-assistant.service.ts

```diff
@@ -13,6 +13,32 @@ export type TypeAnnotationResponse = {
   }>;
 };
 
+export interface UnannotatedArgument {
```

Comment:
make sure to annotate `Readonly` and `ReadonlyArray` so that frontend disallow modifications of data returned from backend.

Reply:
I think you need this format instead.
```
export interface UnannotatedArgument extends Readonly<{
   name: string;
   startLine: number;
   startColumn: number;
   endLine: number;
   endColumn: number;
 }> {}
```